src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -321,11 +321,9 @@ spacetime_deriv_of_power_log_factor_metric_lapse(
 // contain the temporal and spatial roll-on functions.
 template <size_t SpatialDim, typename Frame>
 void damped_harmonic_h(
-    const gsl::not_null<
-        typename db::item_type<Tags::GaugeH<SpatialDim, Frame>>*>
+    const gsl::not_null<db::item_type<Tags::GaugeH<SpatialDim, Frame>>*>
         gauge_h,
-    const typename db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>&
-        gauge_h_init,
+    const db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>& gauge_h_init,
```

Comment:
Write the tensor type explicitly where you know it, instead of resorting to item_type. Helps compile times (I think) and also decouples functions that don‘t need to know anything about databoxes and tags from those that do.